npe2/cli.py

```diff
@@ -231,7 +246,7 @@ def _make_rows(pm_dict: dict, normed_fields: Sequence[str]) -> Iterator[List]:
 @app.command()
 def list(
     fields: str = typer.Option(
-        "name,version,npe2,contributions",
+        "name,version,npe2,contributions,contributions.widgets.display_name",
```

Comment:
i don't think the default value should be changed here.  too specific to a particular use case.  If you want to get all the widget display names, you should use the `fields` option of `npe2 list` manually

Reply:
Yes indeed, I reverted the change :)